### PR TITLE
IIB Release v8.5.0

### PR DESCRIPTION
**Pull request by JAVGan**

### CODE REVIEW

**Comment by JAVGan** [comment]
@lipoja @chandwanitulsi @yashvardhannanavati @zanssa PTAL

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 8.5.0
+* Update dependency cryptography to v41.0.6 by @renovate in https://github.com/release-engineering/iib/pull/595
```
Comment (yashvardhannanavati) [inline CHANGELOG.md:10]: I think a few other changes are missing from here

[Add guardrails for UniqueViolation caused by parallel requests](https://github.com/release-engineering/iib/commit/c6dd3fff0aaccb08d6f2cd7075d0f8cadf1c9897)

[Adding missing $ref for RegenerateBundleRequest](https://github.com/release-engineering/iib/commit/8414d96e1a1eb3529a8b2c9f69dae4e35932bb5f)

[Adding parameter ignore_bundle_ocp_version to merge-index API](https://github.com/release-engineering/iib/commit/bf50d7f103b91ab03bb4dbba616a0306c7cfa746)

**Review by yashvardhannanavati — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (JAVGan) [reply CHANGELOG.md:10]: Thanks!

I'll add

**Review by yashvardhannanavati — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (zanssa) [reply CHANGELOG.md]: This commit [Adding missing $ref for RegenerateBundleRequest](https://github.com/release-engineering/iib/commit/8414d96e1a1eb3529a8b2c9f69dae4e35932bb5f) is still missing in the release notes :)

**Pull request merged**